src/app/todos-data.service.spec.ts

```diff
@@ -33,7 +33,7 @@ describe('HttpClient testing', () => {
 
   it('should work with external service', fakeAsync(() => {
     const { dataService, expectOne, get } = http();
-    get(UserService).getUser.andCallFake(() => {
+    get<SpyObject<UserService>>(UserService).getUser.andCallFake(() => {
```

Comment:
Why do we need to add the `SpyObject` here?

Reply:
Http factory does not support passing `mocks` property. So by default no providers are mocked. That is the reason `SpectatorHTTP` returns just type `S` instead of `SpyOject<S>`.

In this test we explicitly add a `mockProvider` ourselves so we know what we are doing.

I was planning on adding automatic mocking for HTTP as well, I'll get back to you about that.